.github/workflows/docker-image.yml

```
@@ -0,0 +1,24 @@
+    - uses: actions/checkout@v4
+    - name: Build and push the Docker image
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        echo "Building Docker image..."
+        REPO_NAME_LOWER=$(echo "${{ github.repository }}" | tr '[:upper:]' '[:lower:]')
+        docker build . --file Dockerfile --tag ghcr.io/$REPO_NAME_LOWER:latest
+        echo "Logging in to GitHub Packages..."
+        echo ${{ secrets.GITHUB_TOKEN }} | docker login ghcr.io -u ${{ github.actor }} --password-stdin
+        echo "Pushing Docker image..."
+        docker push ghcr.io/$REPO_NAME_LOWER:latest
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)